app/templates/codelijsten/edit.hbs

```diff
@@ -49,15 +49,15 @@
         >
           <AuButton
             @skin="link"
-            @icon="bin"
+            @icon="archive"
             @iconAlignment="left"
             @alert={{true}}
             role="menuitem"
             @disabled={{this.isPrivateConceptScheme}}
-            @loadingMessage={{t "messages.loading.isDeleting"}}
+            @loadingMessage={{t "messages.loading.isArchiving"}}
             {{on "click" (fn (mut this.isDeleteModalOpen) true)}}
           >
-            {{t "crud.delete"}}
+            {{t "crud.archive"}}
           </AuButton>
```

Comment:
What color should the archive button be? It's important to let the user know that this is something serious (red) but on the other hand it is only an archive so it can be blue. Maybe orange?

![image](https://github.com/lblod/frontend-form-builder/assets/36266973/69892c0a-b0cf-49bb-ab14-6cd5c0678983)